easy_context/dist_flash_attn/lightseq_async_attn.py

```diff
@@ -249,8 +249,9 @@ def _lightseq_forward(q, k, v, causal, sm_scale, comm_mode):
     Lq, Lk, Lv = q.shape[-1], k.shape[-1], v.shape[-1]
     assert Lq == Lk and Lk == Lv
     assert Lk in {16, 32, 64, 128}
-    BLOCK_M = 128
-    BLOCK_N = 64
+    # Why do I have to change it from 128 64 to 32 32?
+    BLOCK_M = 32
```

Comment:
Why do I have to change it from 128 64 to 32 32?
If I keep it to 128 64. There will be either illegal memory access complaints from cuda, or not enough resource complaints from triton.
Experiments done on 8 A100.